kubernetes/apps_kubeclient.go

```diff
@@ -153,21 +167,25 @@ func (kc *kubeClient) GetSpace(spaceName string) (*app.SimpleSpace, error) {
 	// https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/space/create/pipelines/pipelines.component.ts
 	buildconfigs, err := kc.getBuildConfigs(spaceName)
 	if err != nil {
-		return nil, err
+		return nil, errs.WithStack(err)
 	}
 
 	// Get all applications in this space using BuildConfig names
-	apps := make([]*app.SimpleApp, 0)
+	apps := []*app.SimpleApp{}
 	for _, bc := range buildconfigs {
 		appn, err := kc.GetApplication(spaceName, bc)
 		if err != nil {
-			return nil, err
+			return nil, errs.WithStack(err)
 		}
 		apps = append(apps, appn)
 	}
 
 	result := &app.SimpleSpace{
-		Applications: apps,
+		Type: "space",
```

Comment:
Is this change in the response format going to be supported by the UI once `wit` is redeployed ?

Reply:
There is already a UI branch which supports this.   The issue is that it's a breaking change, so they both need to be deployed to prod-preview (or prod) at the same time.